app/src/main/java/org/simple/clinic/home/patients/PatientsModule.kt

```diff
@@ -22,4 +22,10 @@ class PatientsModule {
   fun hasUserDismissedApprovedStatus(rxSharedPrefs: RxSharedPreferences): Preference<Boolean> {
     return rxSharedPrefs.getBoolean("approved_status_dismissed", false)
   }
+
+  @Provides
+  @Named("app_update_dialog_shown_at")
```

Comment:
This timestamp exposes a presentation detail that the nudging of app update is done via a dialog. I feel we can make this name a little more generic, something like `app_update_last_nudged_at`. What do you think?

Reply:
Done